CI/travis/before_install_linux

```diff
@@ -11,6 +11,13 @@ handle_centos() {
 	sudo docker pull centos:centos${OS_VERSION}
 }
 
+handle_bionic() {
```

Comment:
you could call this `handle_ubuntu_docker` and it should only run for builds that define `OS_TYPE=ubuntu_docker`
and then you can do `sudo docker pull ubuntu:$OS_VERSION` and you wouldn't need the changes in the `handle_${OS_TYPE}` logic